app/modules/users/resources.py

```diff
@@ -81,7 +95,13 @@ def post(self, args):
         return new_user
 
     @api.login_required(oauth_scopes=['users:write'])
-    @api.permission_required(permissions.AdminRolePermission())
+    @api.permission_required(
+        permissions.ModuleAccessPermission,
+        kwargs_on_request=lambda kwargs: {
+            'module': User,
+            'action': AccessOperation.DELETE,
+        },
+    )
     @api.parameters(parameters.DeleteUserParameters())
     def delete(self, args):
```

Comment:
Can someone explain why Users has a delete operation at the module level rather than just at he UserByID level below?

Reply:
This should be moved.

Reply:
moved or removed?

Reply:
I've moved it to the endpoint where you must pass a UUID at the end.